diff -ruN lazarus/ide/buildlazdialog.pas lazarus.new/ide/buildlazdialog.pas
--- lazarus/ide/buildlazdialog.pas	2019-03-07 00:02:00.000000000 +0100
+++ lazarus.new/ide/buildlazdialog.pas	2019-11-17 20:38:46.477813638 +0100
@@ -510,7 +510,7 @@
       if IdeBuildMode=bmBuild then
         Cmd:='idepkg'
       else
-        Cmd:='cleanide ide';
+        Cmd:='ide';
 
       if (not fOutputDirRedirected) and (not CheckDirectoryWritable(fWorkingDir)) then
         exit(mrCancel);
